src/components/PastTripCard.js

```diff
@@ -1,17 +1,65 @@
 import pastTripList from '../data';
 import styled from 'styled-components';
 import PastTripList from './PastTripList';
+import { useState } from 'react';
+import PastTripForm from './PastTripForm';
+import PastTripsNotesList from './PastTripNotesList';
+
+import { BsArrowLeftCircleFill } from 'react-icons/bs';
+
+export default function PastTripCard({ notes, onHandleNewNote }) {
+  const [active, setActive] = useState(true);
 
-export default function PastTripCard() {
   return (
-    <PastTripCards role="list" aria-label="past-trips">
-      {pastTripList.map(({ country, city, _id }) => {
-        return <PastTripList country={country} city={city} key={_id} />;
-      })}
-    </PastTripCards>
+    <>
+      {active && (
+        <PastTripCards
+          onClick={() => handleCardToggle()}
+          role="list"
+          aria-label="past-trip"
```

Comment:
Since it is a list, wouldn't it be better to use the plural here?
```suggestion
          aria-label="past-trips"
```